### ISSUE TITLE
Mouse wheel to change wepon.

### ISSUE THREAD

**Issue by Lioncky** [nipkownix/re4_tweaks#113]
A count of open inventory , a much alloc. In new hd project it make game crash easier in low device fans.
Why not add a wheel to change wepon, and etc...
Swith any key to select the wheel change object. 
Just advice.

**Comment by Mister-Curious** [comment]
This can easily be accomplished with:
- [Raz0r's Trainer](https://www.nexusmods.com/residentevil4/mods/32)
- [AutoHotKey](https://www.autohotkey.com/)

There is a section in the Trainer called 'Modernizer' which has a 'Fast Equip' function. IRRC the Fast Equip binds to the DPAD somewhere ,but with AHK you could assign mousewheel movement to the DPAD.

**Comment by emoose** [comment]
Unfortunately compatibility with Raz0r's trainer might not be that certain in future, like I found at https://github.com/nipkownix/re4_tweaks/issues/22#issuecomment-1037799043 the trainer seems to overwrite code back to vanilla every frame for some reason, seeing that was kinda worrying since there's a good chance other things that we patch could be getting overwritten by the trainer without us knowing.

Of course we'd like to keep things compatible, but being there's no info about how the trainer/companion DLL works it's not really an easy task for us. I've mentioned on his nexus page that any help from him would be appreciated, but doesn't seem he's very active with RE4 modding right now sadly.

**Comment by Mister-Curious** [comment]
> Unfortunately compatibility with Raz0r's trainer might not be that certain in future, like I found at [#22 (comment)](https://github.com/nipkownix/re4_tweaks/issues/22#issuecomment-1037799043) the trainer seems to overwrite code back to vanilla every frame for some reason, seeing that was kinda worrying since there's a good chance other things that we patch could be getting overwritten by the trainer without us knowing.
> 
> Of course it'd be great to keep things compatible, but seeing as there's no info about how the trainer/companion DLL works it's not exactly an easy task for us. I've mentioned on his nexus page that any help from him would be appreciated, but doesn't seem he's very active with RE4 modding right now sadly.

Yes its annoying when people write things without them being open source and just disappear. Those of us left behind cant fix or change  anything. That being said I would love there to be some kind of attempt for compatibility as the trainer is widely used. Perhaps you could write a patch that prevents his writing to the area of cPlLeon::initCloth & cPlLeon::moveCloth ?
